docs/ios/open-source/manual-instrumentation/error-handling.md

```diff
@@ -0,0 +1,387 @@
+---
+title: Error Handling
+description: Track and manage errors in your iOS app with Embrace
+sidebar_position: 3
+---
+
+# Error Handling
+
+Tracking and handling errors is crucial for understanding problems in your app. Embrace provides comprehensive error tracking capabilities that help you identify and fix issues quickly.
+
+## Types of Errors
+
+Embrace categorizes errors into different types:
+
+- **Crashes**: Unhandled exceptions that terminate your app
+- **Handled Exceptions**: Exceptions that are caught and handled by your code
+- **Error Events**: Error conditions that don't throw exceptions but represent failures
+- **Network Errors**: Failed network requests and API calls
+
+## Tracking Handled Errors
+
+### Basic Error Logging
+
+The simplest way to log an error is with the `logError` method:
```

Comment:
does this method exist in 6x?

<img width="346" alt="Screenshot 2025-04-24 at 8 09 41 AM" src="https://github.com/user-attachments/assets/72125b8d-fe51-491a-9bf4-bd770a7462eb" />